src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.System/Helpers/Icons.cs

```diff
@@ -22,5 +22,5 @@ public static partial class Icons
 
     public static IconInfo ShutdownIcon { get; } = new IconInfo("\uE7E8");
 
-    public static IconInfo SleepIcon { get; } = IconHelpers.FromRelativePaths("Microsoft.CmdPal.Ext.System\\Assets\\sleep.light.png", "Microsoft.CmdPal.Ext.System\\Assets\\sleep.dark.png");
+    public static IconInfo SleepIcon { get; } = new IconInfo("\uE708");
```

Comment:
(the same)

Reply:
Yeah, for those we are moving away from PNGs.. which is great because it the foreground colors weren't correct, didn't adapt to hc and aren't as crisp as fonticons.